php/tests/WellKnownTest.php

```diff
@@ -367,12 +367,28 @@ public function testWrappers()
 
         $m = new Int64Value();
         $m->setValue(1);
-        $this->assertEquals(1, $m->getValue());
+        $this->assertSame(1, $m->getValue());
+
+        $m = new Int64Value();
+        $m->setValue(PHP_INT_MAX);
+        $this->assertEquals(PHP_INT_MAX, $m->getValue());
+
+        $m = new Int64Value();
+        $m->setValue(PHP_INT_MIN);
+        $this->assertEquals(PHP_INT_MIN, $m->getValue());
 
         $m = new UInt64Value();
         $m->setValue(1);
         $this->assertEquals(1, $m->getValue());
 
+        $m = new UInt64Value();
+        $m->setValue(PHP_INT_MAX);
+        $this->assertSame(PHP_INT_MAX, $m->getValue());
+
+        $m = new UInt64Value();
+        $m->setValue('18446744073709551615');
```

Comment:
This line is failing on aarch64

Reply:
@mkruskal-google oh, I see. But that's weird. 
_Works fine on my laptop_ (c)

Tested on MacBook Pro on M1 within PHP 8.2, no errors. Investigating...

Reply:
Just run Ubuntu ARM64 on AWS. Tests passed.
```
Tests: 430, Assertions: 2816, Warnings: 44, Risky: 1.
```

Tested both PHP 8.0 and PHP 8.2 versions.
...

OS info:

```
root@ip-172-31-11-80:~# screenfetch 
                          ./+o+-       root@ip-172-31-11-80
                  yyyyy- -yyyyyy+      OS: Ubuntu 22.04 jammy
               ://+//////-yyyyyyo      Kernel: aarch64 Linux 6.2.0-1015-aws
           .++ .:/++++++/-.+sss/`      Uptime: 0m
         .:++o:  /++++++++/:--:/-      Packages: 641
        o:+o+:++.`..```.-/oo+++++/     Shell: bash 5.1.16
       .:+o:+o/.          `+sssoo+/    Disk: 2.0G / 9.7G (21%)
  .++/+:+oo+o:`             /sssooo.   CPU: 2x ARM Neoverse-N1
 /+++//+:`oo+o               /::--:.   RAM: 336MiB / 1847MiB
```

PHP version:
```
root@ip-172-31-11-80:~# php -v
PHP 8.2.12 (cli) (built: Oct 26 2023 17:33:49) (NTS)
Copyright (c) The PHP Group
Zend Engine v4.2.12, Copyright (c) Zend Technologies
    with Zend OPcache v8.2.12, Copyright (c), by Zend Technologies
```

Reply:
Just realized that I run `composer test` and it passed, but `composer test_c` failed.
Ok, reproduced.